### PR TITLE
Ohie 117 packaging subtask

### PR DESCRIPTION
**Pull request by napiergit**

### CODE REVIEW

**Comment by rcrichton** [comment]
@napiergit Quick note: You may want to edit the readme to reflect the new location of the python script.

**Comment by rcrichton** [comment]
@napiergit I haven't been able to get this to work on a fresh install of the openinfoman. I get the following error:

```
Processing /var/lib/openinfoman/resources/stored_query_definitions/export_organizations_to_csv.xml
PHP Warning:  file_get_contents(/var/lib/openinfoman/resources/stored_query_definitions/export_organizations_to_csv.xml): failed to open stream: No such file or directory in /var/lib/openinfoman/resources/scripts/install_stored_function.php on line 67
PHP Warning:  DOMDocument::loadXML(): Empty string supplied as input in /var/lib/openinfoman/resources/scripts/install_stored_function.php on line 73
PHP Fatal error:  Call to a member function getAttribute() on a non-object in /var/lib/openinfoman/resources/scripts/install_stored_function.php on line 80
+ [[ 255 != 0 ]]
+ exit 1
dpkg: error processing package openinfoman-mapping (--install):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 openinfoman-mapping
```

It seems that `/var/lib/openinfoman/resources/stored_query_definitions/export_organizations_to_csv.xml` doesn't get copied to the target system. Any ideas?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+packaging/.*
+packaging/builds/*
+packaging/targets/*/debian/changelog
```
Comment (rcrichton) [inline .gitignore]: I think we will need the changelog to be committed. Else when someone builds the package the version will be incorrect. We just shouldn't commit changes to the changelog unless we are releasing to the PPA.

I think just ignore anything in packaging/builds

Comment (rcrichton) [inline .gitignore]: Don't worry, I'l make this change as it's small and I need to upload to the OpenHIE PPA anyways.

Comment (napiergit) [inline .gitignore]: Cool thanks.

**Comment by rcrichton** [comment]
:+1:

**Pull request merged**